app/controllers/MissionController.scala

```diff
@@ -108,20 +128,44 @@ class MissionController @Inject() (implicit val env: Environment[User, SessionAu
     )
   }
 
+  def ~=(x: Double, y: Double, precision: Double) = { // Approximate equality check for Doubles
```

Comment:
If you got this suggestion from stackoverflow or something, just add a link to it as well.